--- apertium-3.5.2/apertium/Makefile.am~	2018-08-04 17:52:54.000000000 +0200
+++ apertium-3.5.2/apertium/Makefile.am	2020-03-23 17:37:06.996686819 +0100
@@ -237,7 +237,7 @@
 	       apertium-tmxbuild \
 	       apertium-transfer
 
-bin_SCRIPTS =  $(GENERATEDSCRIPTS) apertium-get/apertium-get
+bin_SCRIPTS =  $(GENERATEDSCRIPTS)
 
 # Workaround for git-svn clone not getting externals; should do
 # nothing in normal SVN checkouts
