--- apertium-3.2.0/configure.ac.wiget	2012-03-30 18:46:44.322626887 +0200
+++ apertium-3.2.0/configure.ac	2012-03-30 18:47:01.332626607 +0200
@@ -122,7 +122,7 @@ AC_CHECK_FUNCS(strcasecmp)
 if test x$(uname) != xDarwin;
 then 
 AC_CHECK_HEADER(pcreposix.h,
-  AC_CHECK_LIB(pcre, pcre_info,[
+  AC_CHECK_LIB(pcre, pcre_fullinfo,[
   LIBS="$LIBS -lpcreposix -lpcre"
   no_comp_check=yes],
   AC_MSG_ERROR([*** unable to locate pcre library ***])),
