--- apertium-3.9.12/apertium/Makefile.am~	2025-03-26 10:26:26.000000000 +0100
+++ apertium-3.9.12/apertium/Makefile.am	2025-08-08 02:41:44.489995872 +0200
@@ -447,49 +447,49 @@
 
 apertium-validate-tagger: Makefile.am validate-header.sh
 	@echo "Creating apertium-validate-tagger script"
-	@echo "#!/usr/bin/env bash" > $@
+	@echo "#!/bin/bash" > $@
 	@cat validate-header.sh >> $@
 	@echo "xmllint --dtdvalid \"$(apertiumdir)\"/tagger.dtd --noout \"\$$FILE1\"" >>$@
 	@chmod a+x $@
 
 apertium-validate-transfer: Makefile.am validate-header.sh
 	@echo "Creating apertium-validate-transfer script"
-	@echo "#!/usr/bin/env bash" > $@
+	@echo "#!/bin/bash" > $@
 	@cat validate-header.sh >> $@
 	@echo "xmllint --dtdvalid \"$(apertiumdir)\"/transfer.dtd --noout \"\$$FILE1\"" >>$@
 	@chmod a+x $@
 
 apertium-validate-interchunk: Makefile.am validate-header.sh
 	@echo "Creating apertium-validate-interchunk script"
-	@echo "#!/usr/bin/env bash" > $@
+	@echo "#!/bin/bash" > $@
 	@cat validate-header.sh >> $@
 	@echo "xmllint --dtdvalid \"$(apertiumdir)\"/interchunk.dtd --noout \"\$$FILE1\"" >>$@
 	@chmod a+x $@
 
 apertium-validate-postchunk: Makefile.am validate-header.sh
 	@echo "Creating apertium-validate-postchunk script"
-	@echo "#!/usr/bin/env bash" > $@
+	@echo "#!/bin/bash" > $@
 	@cat validate-header.sh >> $@
 	@echo "xmllint --dtdvalid \"$(apertiumdir)\"/postchunk.dtd --noout \"\$$FILE1\"" >>$@
 	@chmod a+x $@
 
 apertium-validate-acx: Makefile.am validate-header.sh
 	@echo "Creating apertium-validate-acx script"
-	@echo "#!/usr/bin/env bash" > $@
+	@echo "#!/bin/bash" > $@
 	@cat validate-header.sh >> $@
 	@echo "xmllint --relaxng \"$(prefix)\"/share/lttoolbox/acx.rng --schema \"$(prefix)\"/share/lttoolbox/acx.xsd --noout \"\$$FILE1\"" >>$@
 	@chmod a+x $@
 
 apertium-validate-crx: Makefile.am validate-header.sh
 	@echo "Creating apertium-validate-crx script"
-	@echo "#!/usr/bin/env bash" > $@
+	@echo "#!/bin/bash" > $@
 	@cat validate-header.sh >> $@
 	@echo "xmllint --dtdvalid \"$(apertiumdir)\"/crx.dtd --noout \"\$$FILE1\"" >>$@
 	@chmod a+x $@
 
 apertium-validate-modes: Makefile.am validate-header.sh
 	@echo "Creating apertium-validate-modes script"
-	@echo "#!/usr/bin/env bash" > $@
+	@echo "#!/bin/bash" > $@
 	@cat validate-header.sh >> $@
 	@echo "xmllint --dtdvalid \"$(apertiumdir)\"/modes.dtd --noout \"\$$FILE1\"" >>$@
 	@chmod a+x $@
@@ -497,7 +497,7 @@
 
 apertium-validate-dictionary: Makefile.am validate-header.sh
 	@echo "Creating apertium-validate-dictionary script"
-	@echo "#!/usr/bin/env bash" > $@
+	@echo "#!/bin/bash" > $@
 	@cat validate-header.sh >> $@
 	@echo "# xsd is a non-final command, so just treated as a warning when compiling:" >> $@
 	@echo "xmllint --schema \"$(prefix)\"/share/lttoolbox/dix.xsd --noout \"\$$FILE1\" 2>&1 | grep -ve ' validates$$' -e ' fails to validate$$'" >> $@
@@ -506,7 +506,7 @@
 
 apertium-gen-deformat: Makefile.am deformat-header.sh
 	@echo "Creating apertium-gen-deformat script"
-	@echo "#!/usr/bin/env bash" > $@
+	@echo "#!/bin/bash" > $@
 	@cat deformat-header.sh >> $@
 	@echo "xmllint --dtdvalid \"$(apertiumdir)\"/format.dtd --noout \"\$$FILE1\" && \\" >> $@
 	@if [ `basename xsltproc` == xsltproc ]; \
@@ -520,7 +520,7 @@
 
 apertium-gen-reformat: Makefile.am gen-header.sh
 	@echo "Creating apertium-gen-reformat script"
-	@echo "#!/usr/bin/env bash" > $@
+	@echo "#!/bin/bash" > $@
 	@cat gen-header.sh >> $@
 	@echo "xmllint --dtdvalid \"$(apertiumdir)\"/format.dtd --noout \"\$$FILE1\" && \\" >> $@
 	@echo "xsltproc \"$(apertiumdir)\"/reformat.xsl \"\$$FILE1\" >/tmp/\$$\$$.reformat.l && \\" >> $@
@@ -531,13 +531,13 @@
 
 apertium-utils-fixlatex: Makefile.am utils-fixlatex-header.sh
 	@echo "Creating apertium-utils-fixlatex script"
-	@echo "#!/usr/bin/env bash" > $@
+	@echo "#!/bin/bash" > $@
 	@cat utils-fixlatex-header.sh >> $@
 	@chmod a+x $@
 
 apertium: Makefile.am apertium-header.sh
 	@echo 'Creating apertium script'
-	@echo '#!/usr/bin/env bash' > $@
+	@echo '#!/bin/bash' > $@
 	@echo 'APERTIUM_PATH="$${APERTIUM_PATH:-$(prefix)/bin}"' >>$@
 	@echo 'APERTIUM_DATADIR="$${APERTIUM_DATADIR:-$(prefix)/share/apertium}"' >>$@
 	@echo 'apertium_version="Apertium $(VERSION)"' >>$@
@@ -546,7 +546,7 @@
 
 apertium-unformat: Makefile.am apertium-unformat-header.sh
 	@echo "Creating apertium-unformat script"
-	@echo "#!/usr/bin/env bash" > $@
+	@echo "#!/bin/bash" > $@
 	@echo "APERTIUM_PATH=\"$(prefix)/bin\"" >>$@
 	@cat apertium-unformat-header.sh >>$@
 	@chmod a+x $@
